zuul.d/jobs-ansible-lint.yaml

```
@@ -5,7 +5,8 @@
     name: ansible-lint-tox-docs
     parent: ansible-tox-docs
     vars:
-      sphinx_build_dir: .tox/docs_html
+      # zuul expects a `html` folder to be created under this
+      sphinx_build_dir: .tox/docs
```

```suggestion
      sphinx_build_dir: .tox/docs/html
```
(as discussed on freenode)